spec.md

```diff
@@ -1540,6 +1624,17 @@ message ControllerServiceCapability {
       // Indicates the SP supports the
       // ListVolumesResponse.entry.published_nodes field
       LIST_VOLUMES_PUBLISHED_NODES = 10;
+
+      // Indicates the SP supports the
+      // ListVolumesResponse.entry.volume_health field
+      LIST_VOLUMES_VOLUME_HEALTH = 11;
```

Comment:
instead of adding new capabilities here, the Alpha API proposal suggests a separate AlphaFeature bucket. See https://github.com/container-storage-interface/spec/pull/365/commits/105831102506ec4999e8386c08a3421bed0b6d9f#diff-308dbd5d943fd4b49fc73f1092c253abR948

Reply:
Done